### DISCUSSION TITLE
Midi support

### DISCUSSION THREAD

**Discussion by LouDnl** [LouDnl/USBSID-Pico#2] [Q&A] [answered]
Any questions regarding Midi support can be asked here

**Comment by tobozo** [comment]
hi,

if I plug 2x FPGASID on the usbsid-pico and set them both as dual SID, can I get 4x3 simultaneous notes ?

**Reply by LouDnl** [reply]
In theory with the polyfonic notes patch yes, if you hookup the yellow wire from each fpgasid to the two A5 pins. Then go to the config page and set 4x sid or send '\x42\0\0\0\0' to the ACM port which does the same.

**Reply by LouDnl** [reply, marked as answer]
![Screenshot_20241126_131157_Firefox](https://github.com/user-attachments/assets/9f07e2a8-af45-401f-98e1-fa28db9a83ba)
